text/stacks/0004-jammy-jellyfish.md

```diff
@@ -0,0 +1,106 @@
+
+### Stack IDs
+
+Each of these stacks will declare their stack ID as `io.paketo.stack.jammy`.
+This choice differs from Bionic in that we will **NOT** differentiate between
+the `tiny` and `base/full` variants using the stack ID.
+
+### User IDs
+
+These stacks will differ from Bionic with regards to their UID definitions.
+Each variant will ensure that the UID for the `build` phase is different than
+the `run` phase. This change will align more closely with the
+[recommendations](https://github.com/buildpacks/rfcs/blob/main/text/0085-run-uid.md)
+outlined in the Buildpacks Specification.
+
+### Image Naming and Tagging
+
+The stacks will name and tag their release images with the following pattern:
+
+```
+paketobuildpacks/jammy-{phase}-{variant}:{version}
+```
+
+For example we could see the following images for Jammy stacks:
+
+* `paketobuildpacks/jammy-build-base:latest`
+* `paketobuildpacks/jammy-run-tiny:1.2.3`
+
+This choice is different than the choice made with Bionic to better align the
+image repository references with their logical stack definition. This means
+that, with this naming scheme, it will be much more reasonable to understand
+what is stored at the repository reference
+`paketobuildpacks/jammy-run-base:latest`.
+
+### SBOM
+
+This stack will **NOT** include an SBOM of the form outlined in Stacks
+[RFC0001](https://github.com/paketo-buildpacks/rfcs/blob/main/text/stacks/0001-stack-package-metadata.md).
+This choice is to ready ourselves to adopt whatever Stack SBOM functionality is
+specified by the Cloud Native Buildpacks project while ensuring we don't
+perpetuate the use of our own format for the entirety of this stack's support
+lifetime.
+
+This stack will adopt whatever official SBOM support is declared upstream by
+the CNB project.
+
+## Rationale and Alternatives
+
+We should at least discuss what it would mean to provide a Debian stack. There
+are [LTS releases](https://wiki.debian.org/LTS) for Debian, but they follow a
+different lifecycle and may not provide enough overlap to allow end-users to
+migrate completely.
+
+## Implementation
+
+After the [final
+release](https://discourse.ubuntu.com/t/jammy-jellyfish-release-schedule/23906)
+date on April 21, 2022, we can release our official stack versions.
+
+### Repositories
+
+The Stacks subteam will create 3 new repos for Jammy:
+
+* `jammy-base-stack`
+* `jammy-full-stack`
+* `jammy-tiny-stack`
+
+Each of these repos will contain the configuration for its variant of the stack
+as well as the releases and their related artifacts.
+
+## Prior Art
+
+* [`stacks`](https://github.com/paketo-buildpacks/stacks)
+* [`base-stack-release`](https://github.com/paketo-buildpacks/base-stack-release)
+* [`full-stack-release`](https://github.com/paketo-buildpacks/full-stack-release)
+* [`tiny-stack-release`](https://github.com/paketo-buildpacks/tiny-stack-release)
+
+## Unresolved Questions and Bikeshedding
+
+* What does this mean for builders?
```

Comment:
I would imagine we'd ship another set of builders for the jammy stacks. @paketo-buildpacks/builders-maintainers thoughts?

Reply:
As it stands, we release 6 builders (tiny, base, full, and buildpackless varieties of these). If we duplicate these efforts for jammy, we'll be on the hook for 12 builders in total. That feels like a lot, but perhaps makes sense until Bionic EOLs next year. 

Since builders aren't needed in perpetuity for rebasing, perhaps once bionic is EOL, the current builder repositories are switched from publish bionic to jammy builders?

There's also the topic of repo and tag naming, which will need to be ironed out. Would we do something like:
`paketobuildpacks/builder:<version>-<buildpackless?>-<OS>-<variant>`
`paketobuildpacks/builder:1.2.3-buildpackless-jammy-full`

Reply:
Is this a point we should be aligned on for this RFC? If so, is @fg-j suggestion the right path forward here? I can't remember what we discussed around this point at the working group meeting. Would not having jammy builders until next year be a hindrance to users?

Reply:
I don't think this is in scope for this RFC. We will need to figure this out, and so I wanted builder maintainers to provide some input, but I don't think we need to actually decide the names for builders for this RFC to pass.